lib/smart_proxy_remote_execution_ssh/runners/script_runner.rb

```diff
@@ -185,7 +185,7 @@ def prepare_start
       @pid_path = File.join(File.dirname(@remote_script), 'pid')
       su_method = @user_method.instance_of?(SuUserMethod)
       wrapper = <<~SCRIPT
-        if [ "$1" == "inner" ]; then
+        if [ "$1" = "inner" ]; then
```

Comment:
With `==` you get
```
[: inner: unexpected operator
```
and the `if` fails, executing the `else` part (with the `exec`) again and again and again in a loop.